<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDialog</class>
 <widget class="QDialog" name="NewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>285</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>812</width>
    <height>285</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>812</width>
    <height>285</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New</string>
  </property>
  <widget class="QStackedWidget" name="StackedWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>0</y>
     <width>801</width>
     <height>281</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="NewItemLayout">
        <item>
         <widget class="QTreeWidget" name="ProjectTree">
          <column>
           <property name="text">
            <string>Project</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="FileTree">
          <column>
           <property name="text">
            <string>File</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="ExtensionLabel">
          <property name="text">
           <string>File extension(optional):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>200</y>
       <width>381</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QFrame" name="ExtensionFrame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLineEdit" name="ExtensionEdit">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>381</width>
           <height>36</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="SubmitLayout">
        <item>
         <widget class="QPushButton" name="CancelButton">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="NextButton">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="DescripionLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>391</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Description: </string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>DescripionLabel</zorder>
    <zorder>verticalLayoutWidget_2</zorder>
    <zorder>verticalLayoutWidget_3</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="FileNameLabel">
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>File Name:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NameEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="FileNameLabel_2">
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>2</number>
        </property>
        <property name="text">
         <string>Location:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="LocationEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="BrowseButton">
          <property name="text">
           <string>Browse..</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="BackButton">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="CreateButton">
          <property name="text">
           <string>Create File</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
